src/main/java/uk/gov/hmcts/reform/pcqbackend/security/SecurityConfiguration.java

```diff
@@ -1,57 +1,46 @@
 package uk.gov.hmcts.reform.pcqbackend.security;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.core.annotation.Order;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.builders.WebSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
 import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
 import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
 
-import java.security.Security;
 import javax.servlet.http.HttpServletResponse;
+import java.security.Security;
 
 @Configuration
 @EnableWebSecurity
-@Order(1)
-public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+@SuppressWarnings({"PMD.SignatureDeclareThrowsException"})
+
+public class SecurityConfiguration {
```

Comment:
Please add public **void configure(WebSecurity web)** back. As already mentioned, I'd rather leave this here for now and remove this configuration as part of a separate ticket.

Reply:
Added this back